ssv/spectest/tests/runner/construction/many_shares.go

```diff
@@ -0,0 +1,29 @@
+package runnerconstruction
+
+import (
+	"github.com/ssvlabs/ssv-spec/ssv/spectest/tests"
+	"github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv-spec/types/testingutils"
+)
+
+func ManyShares() tests.SpecTest {
+
+	ks := testingutils.KeySetMapForValidators(10)
+	shares := testingutils.ShareMapFromKeySetMap(ks)
+
+	// No errors since one share must be valid for all runners
```

Comment:
Is this comment right for this test?

Reply:
Indeed not. Updating. Thanks for the catch!